examples/minimal/variables.tf

```diff
@@ -16,3 +16,13 @@ variable "tags" {
     Environment = "Example"
   }
 }
+
+variable "ingress_protocol" {
+  type        = string
+  description = "Protocol for ingress rules. If not icmp, icmpv6, tcp, udp, or all use the protocol number."
+}
+
+variable "egress_protocol" {
+  type        = string
+  description = "Protocol for egress rules. If not icmp, icmpv6, tcp, udp, or all use the protocol number."
+}
```

Comment:
I'd prefer not to duplicate variables in the examples unless absolutely necessary. Let's just go with a realistic default within the example.

Reply:
We will keep it simple without variables just for the examples.